### ISSUE TITLE
Docstrings not available in python bindings

### ISSUE THREAD

**Issue by pietrolesci** [huggingface/tokenizers#493]
Hi all,

Thanks a lot for this amazing library!

I am having issues with the docstings. I noticed that docstrings do not get propagated. For example although there is a docstring [here](https://github.com/huggingface/tokenizers/blob/master/bindings/python/py_src/tokenizers/trainers/__init__.pyi) for the trainers, when running

```python
from tokenizers.trainers import BpeTrainer
?BpeTrainer
```
in a Jupyter notebook, VScode or Colab, returns 

```
Init signature: BpeTrainer(self, /, *args, **kwargs)
Docstring:      <no docstring>
Type:           type
Subclasses:     
```

**Details**: tokenizers v0.9.3

**Comment by n1t0** [comment]
Indeed, but the `.pyi` files only seem to be used by some IDE like `PyCharm` or `VSCode`. We started updating the "real" docstrings (ie `__doc__` in Python) to have them propagated properly in the `documentation` branch, but it does not cover everything yet. It will be way better in the next release anyway!

**Comment by pietrolesci** [comment]
Hi @n1t0,
Thanks a lot for your reply. I am _really_ looking forward to the new release! Btw, great job so far! :)

**Comment by n1t0** [comment]
We just released the Python v0.10.0 which include docstrings as discussed. Closing this issue.
